src/backend/joanie/core/models/products.py

```diff
@@ -202,42 +197,25 @@ def synchronize_products(products, visibility=None):
                         {**course_run_dict, "course": course.code}
                     )
 
+        return equivalent_course_runs
+
+    @staticmethod
+    def synchronize_products(products, visibility=None):
+        """
+        Synchronize a product's related course runs by calling remote web hooks.
+
+        visibility: [CATALOG_VISIBILITY_CHOICES]:
+            If not None, force visibility for the synchronized products. Useful when
+            synchronizing a product that does not have anymore course runs and should
+            therefore be hidden.
+        """
+
+        equivalent_course_runs = Product.get_synchronize_products(products, visibility)
```

Comment:
I wouldn't split this in another method as it makes it harder to understand and doesn't bring any benefit as it is not going to be called from anywhere else? If I'm missing something and this factorization is necessary, call it `get_equivalent_course_runs_for_products`?